scripts/sort-makefile-am-sources.pl

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/perl
+
```

Comment:
AFAIK, these are pretty much mandatory in any non-trivial Perl script:

```suggestion

use strict;
use warnings;
```

Reply:
adding